src/bufr/CMakeLists.txt

```diff
@@ -67,8 +67,8 @@ ecbuild_add_library( TARGET   ingester
 
 add_library( ${PROJECT_NAME}::ingester ALIAS ingester )
 
-ecbuild_add_executable( TARGET  convertToIoda.x
-                        SOURCES main.cpp
+ecbuild_add_executable( TARGET  bufr2ioda
```

Comment:
I think this was @aerorahul , but I think this should end in .x to be consistent with other JEDI applications (such as fv3-jedi, etc.).

Reply:
done.